File: JavaBasic/153_array_avg.java
// 153	153_array_avg.java	Find the average of all elements in a given array.
public class _153_array_avg {
    public static void main(String[] args) {
        int[] arr = {10, 20, 30, 40, 50};
        int sum = 0;
        for (int x : arr) sum += x;
        System.out.println("Average: " + (double)sum/arr.length);
    }
}

File: JSBASIC/153_array_avg.js
// 153	153_array_avg.js	Find the average of all elements in a given array.

let arr = [10, 20, 30, 40, 50];
let sum = 0;
for (let i = 0; i < arr.length; i++) {
    sum += arr[i];
}
let avg = sum / arr.length;
console.log("Array:", arr);
console.log("Average:", avg);
